spec/models/admin_set_spec.rb

```diff
@@ -76,16 +76,64 @@
     expect(subject.reload.description).to eq ["description"]
   end
 
+  describe "#default_set?" do
+    context "with default AdminSet" do
+      before do
+        subject.id = described_class::DEFAULT_ID
+        subject.save!
+      end
+      it "returns true" do
+        expect(subject.default_set?).to be true
```

Comment:
Check out RSpec's sweet predicate matchers:

```ruby
expect(subject).to be_default_set
```

Or you can even one-line this:

```ruby
it { is_expected.to be_default_set }
```